backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/trace/HUTraceEventsService.java

```diff
@@ -225,6 +217,11 @@ void createAndAddEventsForInventoryLines(
 			for (final InventoryLineHU inventoryLineHU : inventoryLineHUs)
 			{
 				final HuId huId = inventoryLineHU.getHuId();
+				if(huId == null)
```

Comment:
this one can't really happen, if it does I believe we should throw an error as we would be missing the events for that inventory like see: 

![image](https://github.com/user-attachments/assets/8ba9e5f0-e687-46c0-8a47-79edcc8ae531)